umh.docs.umh.app/static/js/requirements.js

```
@@ -1,164 +1,174 @@
-  
-        // Inject system requirements into #requirements-2
-        const requirementsDiv2 = document.getElementById('requirements-2');
-        if (requirementsDiv2) {
-          requirementsDiv2.innerHTML = `
+      }
+
+      const releaseVersion = latestRelease.releaseVersion;
+
+      // Create a list for system requirements
+      const systemRequirements = [
+        { name: 'CPU', value: latestRelease.systemRequirements.cpu.cores + ' Cores' },
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)